### DISCUSSION TITLE
Invalid XML character

### DISCUSSION THREAD

**Discussion by jtyme** [nextgenhealthcare/connect#6014] [Q&A]
Trying to see what is best practice for handling invalid xml character (Unicode 0x3) was found in the element content.

Currently we are using the following regex 
var newMessage = message.replace(/[\x00-\x08]|[\x0B-\x0C]|[\x0E-\x1F]/g, '');
And this works.

But we are using it in the Custom Attachment Javascript section. Is there a better place or process to resolve this? We can't reach out to the source of the message and need to resolve this on our end.

**Comment by pacmano1** [comment]
The preprocesor script might be the typical place peope do this.
